### PR TITLE
feat(*): set-up my identity documents section

### PR DESCRIPTION
**Pull request by anooparveti**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/acquisition-storage`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-11202
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~ [n/a]

## Description

Add "My Identity Documents" section in Manager

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.controller.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default class AccountUserIdentityDocumentsController {
+  /* @ngInject */
+}
```
Comment (mohammed-zahaf) [inline packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.controller.js:3]: Empty controller can be delete.

Comment (anooparveti) [reply packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.controller.js:3]: The scope of this PR is to set-up an empty page. We have a different ticket to complete the page.

File: packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<h2 data-translate="user_account_identity_documents"></h2>
```
Comment (mohammed-zahaf) [inline packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.html:1]: may be having p with h2 style is better ?
```suggestion
<p class="oui-heading_2" data-translate="user_account_identity_documents"></p>
```

File: packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.module.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+angular
+  .module(moduleName, [
+    ngAtInternet,
+    ngOvhHttp,
+    'oui',
+    'pascalprecht.translate',
+    'ui.router',
+  ])
+  .component('accountUserIdentityDocuments', component)
```
Comment (mohammed-zahaf) [inline packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.module.js:17]: prefixing with app related scope could be good:
```suggestion
  .component('dedicatedAccountUserIdentityDocuments', component)
```

File: packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.module.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import ngAtInternet from '@ovh-ux/ng-at-internet';
+import ngOvhHttp from '@ovh-ux/ng-ovh-http';
+
+import component from './user-identitiy-documents.component';
+import routing from './user-identity-documents.routes';
+
+const moduleName = 'UserAccountIdentityDocuments';
```
Comment (mohammed-zahaf) [inline packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.module.js:7]:
```suggestion
const moduleName = 'DedicatedUserAccountIdentityDocuments';
```

File: packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.controller.js (hunk 1 above)
Comment (mohammed-zahaf) [reply packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.controller.js:3]: Ah ok, on my side i have habit to let a todo comment, to inform others about the incoming tasks.
```suggestion
/**TODO: controller will be completed within incoming tasks*/
export default class AccountUserIdentityDocumentsController {
  /* @ngInject */
}
```

File: packages/manager/apps/container/src/container/legacy/server-sidebar/universe/AccountSidebar.tsx [hunk 5]
```diff
@@ -40,6 +40,24 @@ export default function AccountSidebar() {
       routeMatcher: new RegExp('^/useraccount'),
     });
 
+    const featureAvailablity = await reketInstance.get(
```
Comment (sachinrameshn) [inline packages/manager/apps/container/src/container/legacy/server-sidebar/universe/AccountSidebar.tsx:43]: Spell Mistake:  featureAvailablity -> featureAvailability

**Review by frenautvh — APPROVED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Pull request merged**